cachito/web/models.py

```diff
@@ -286,6 +286,7 @@ class Request(db.Model):
     """A Cachito user request."""
 
     id = db.Column(db.Integer, primary_key=True)
+    created = db.Column(db.DateTime(), nullable=False, index=True, default=sqlalchemy.func.now())
```

Comment:
Can you tell me which postgresql datatype is this, pls?

Reply:
Timestamp without timezone:
```
                                         Table "public.request"
       Column       |            Type             |                      Modifiers                       
--------------------+-----------------------------+------------------------------------------------------
 id                 | integer                     | not null default nextval('request_id_seq'::regclass)
 repo               | character varying           | not null
 ref                | character varying           | not null
 user_id            | integer                     | 
 submitted_by_id    | integer                     | 
 request_state_id   | integer                     | 
 packages_count     | integer                     | 
 dependencies_count | integer                     | 
 created            | timestamp without time zone | 
Indexes:
    "request_pkey" PRIMARY KEY, btree (id)
    "ix_request_request_state_id" UNIQUE, btree (request_state_id)
    "ix_request_created" btree (created)
    "ix_request_ref" btree (ref)
    "ix_request_repo" btree (repo)
```